intelligence-services/latest/using/index.md

```diff
@@ -94,15 +94,23 @@ If you're planning to use custom recognizers, custom classifiers, or custom meta
 
         See [Custom AI rendition definitions]({% link intelligence-services/latest/config/comprehend.md %}#custom-ai-rendition-definitions) for configured renditions.
 
-    3. For custom metadata extraction (using Textract), request a rendition and add a custom AI aspects. For example:
+    3. For custom metadata extraction (using Textract), request a rendition and add custom AI aspects. For example:
 
-        * Request AI renditions: `aiTextract`
+        * Request AI rendition: `aiTextract`
         * Add aspect: `AI Text Lines`
         * Add aspect: `Applicant Info`
         * Add aspect: `w9form`
 
         See [Custom AI content model (Textract)]({% link intelligence-services/latest/config/textract.md %}#custom-ai-content-model) for configured aspects.
 
+    4. For speech to text extraction (using Transcribe), request renditions and add a custom AI aspect. For example:
+
+        * Request AI rendition: `aiSpeechToText`
+        * Request AI rendition: `webvtt`
+        * Add aspect: `AI Transcript`
+
+        See [Custom AI content model (Transcribe)]({% link intelligence-services/latest/config/transcribe.md %}#custom-ai-content-model) for configured aspects.
+
```

Comment:
You have already given an example of rendition, you don't need this, otherwise, you should add an example of all the available renditions :)